### ISSUE TITLE
[LightGBM] [Warning] Unknown parameter: NA

### ISSUE THREAD

**Issue by dixit16** [Microsoft/LightGBM#1316]
Hi ,

I am trying to train and test a simple light-gbm model and while running predict I am getting the following error : [LightGBM] [Warning] Unknown parameter: NA.

Below is the code snippet that I am using -

I have kept train and test data same in order to figure out the issue -

data_train = as.matrix(select(small_train,-c(ip,click_day)))
data_test = as.matrix(select(small_train,-c(ip,click_day,is_attributed)))

bst <- lightgbm(data = data_train,
                label = small_train$is_attributed,
                num_leaves = 4,
                learning_rate = 1,
                nrounds = 2,
                objective = "binary")
pred <- predict(bst, data_test)

**Comment by guolinke** [comment]
@dixit16 please try this https://github.com/Microsoft/LightGBM/issues/571#issuecomment-305331818

@Laurae2 any other suggestions?

**Comment by guolinke** [comment]
@dixit16 where this warning happen ? when performing predict ?

**Comment by guolinke** [comment]
@dixit16  you can try #1323

**Comment by dixit16** [comment]
@guolinke : Hey , sorry couldn'y respond earlier. Yes the warning shows up while predicting. I am not sure of the impact of this warning on prediction ..
